rtf_vis_tool/src/Components/ImageViewer.js

```diff
@@ -0,0 +1,55 @@
+/*
+Author: Hayk Stepanyan
+*/
+import React from "react";
+
+import '../stylesheets/ImageViewer.css';
+import imageShapes from '../images/image_shapes.json';
+
+
+function ImageViewer(props) {
+
+    // import image files
+    function importAll(r) {
+        let images = {};
+        r.keys().map((item, index) => { images[item.replace('./', '')] = r(item); });
+        return images;
+      }
+    const images = importAll(require.context('../images', false, /\.(png|jpe?g|svg)$/));
+    const imageFileNames = Object.keys(images);
+
+    // sort file names
+    imageFileNames.sort((a, b) => {
+        a = parseInt(a.split(".")[0])
+        b = parseInt(b.split(".")[0])
+        console.log(a)
+        console.log(b)
```

Comment:
do we need to log here?

Reply:
thanks for the catch, it was for debugging purposes, removed in the new commit